Error when try test in my fork

https://github.com/BataevDaniil/restic/actions/runs/27065575907/job/79885745422#logs
```
-- FAIL: TestRepositoryIncrementalIndex (0.09s)
    --- FAIL: TestRepositoryIncrementalIndex/v2 (0.05s)
        testing.go:29: using low-security KDF parameters for test
        config.go:60: disabling check of the chunker polynomial
        repository_test.go:375: unexpected error: Detected data corruption while saving blob e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855: hash mismatch
            Corrupted blobs are either caused by hardware issues or software bugs. Please open an issue at https://github.com/restic/restic/issues/new/choose for further troubleshooting.
            
        repository_test.go:366: unexpected error: context canceled
            
FAIL
```

--- a/go.mod
+++ b/go.mod
@@ -22,8 +22,8 @@ require (
 	github.com/go-ole/go-ole v1.3.0
 	github.com/google/go-cmp v0.7.0
 	github.com/hashicorp/golang-lru/v2 v2.0.7
-	github.com/klauspost/compress v1.18.6
-	github.com/minio/minio-go/v7 v7.2.0
+	github.com/klauspost/compress v1.18.4
+	github.com/minio/minio-go/v7 v7.1.0
 	github.com/ncw/swift/v2 v2.0.5
 	github.com/peterbourgon/unixtransport v0.0.7
 	github.com/pkg/errors v0.9.1
@@ -65,6 +65,7 @@ require (
 	github.com/envoyproxy/protoc-gen-validate v1.3.3 // indirect
 	github.com/felixge/fgprof v0.9.3 // indirect
 	github.com/felixge/httpsnoop v1.0.4 // indirect
+	github.com/go-ini/ini v1.67.0 // indirect
 	github.com/go-jose/go-jose/v4 v4.1.4 // indirect
 	github.com/go-logr/logr v1.4.3 // indirect
 	github.com/go-logr/stdr v1.2.2 // indirect
@@ -104,5 +105,4 @@ require (
 	google.golang.org/genproto/googleapis/rpc v0.0.0-20260523011958-0a33c5d7ca68 // indirect
 	google.golang.org/grpc v1.81.1 // indirect
 	google.golang.org/protobuf v1.36.11 // indirect
-	gopkg.in/ini.v1 v1.67.2 // indirect
 )

--- a/internal/repository/repository_test.go
+++ b/internal/repository/repository_test.go
@@ -80,6 +80,28 @@ func testSave(t *testing.T, version uint, calculateID bool) {
 	}
 }
 
+func TestSaveLoadZeroSizedBlob(t *testing.T) {
+	repository.TestAllVersions(t, testSaveLoadZeroSizedBlob)
+}
+
+func testSaveLoadZeroSizedBlob(t *testing.T, version uint) {
+	repo, _, _ := repository.TestRepositoryWithVersion(t, version)
+
+	var data []byte
+	id := restic.Hash(data)
+
+	rtest.OK(t, repo.WithBlobUploader(context.TODO(), func(ctx context.Context, uploader restic.BlobSaverWithAsync) error {
+		sid, _, _, err := uploader.SaveBlob(ctx, restic.DataBlob, data, id, false)
+		rtest.OK(t, err)
+		rtest.Equals(t, id, sid)
+		return nil
+	}))
+
+	buf, err := repo.LoadBlob(context.TODO(), restic.DataBlob, id, nil)
+	rtest.OK(t, err)
+	rtest.Equals(t, 0, len(buf))
+}
+
 func TestSavePackMerging(t *testing.T) {
 	t.Run("75%", func(t *testing.T) {
 		testSavePackMerging(t, 75, 1)

